docs/packages/compression.rst

```diff
@@ -1,8 +1,8 @@
 Compression
 -----------
 
- * `BZip2 <https://github.com/ruslo/hunter/wiki/pkg.bzip2>`_ - high-quality data compressor.
+ * :ref:`pkg.BZip2` - high-quality data compressor.
  * `minizip <https://github.com/ruslo/hunter/wiki/pkg.minizip>`_ - enables to extract files from a .zip archive file.
```

Comment:
Should it be
```
:ref:`pkg.minizip`
```
now?

Reply:
Yes. The labels are created to have exactly the name the folder in `cmake/projects` has. When having `cmake/projects/LibFoo` I've generated a label `pkg.LibFoo`